keras/keras_simple.py

```diff
@@ -87,7 +87,9 @@ def objective(trial):
         "There is now only one Keras: tf.keras. "
         "There may be some breaking changes for some workflows by upgrading to keras 2.4.0. "
         "Test before upgrading. "
-        "REF:https://github.com/keras-team/keras/releases/tag/2.4.0"
+        "REF: https://github.com/keras-team/keras/releases/tag/2.4.0. "
+        "There is an alternative callback function that can be used instead: "
+        ":class:`~optuna.integration.TFKerasPruningCallback`",
```

Comment:
This is not an pruning/callback example but should we still keep this addition?

Reply:
Thank you for your feedback! I agree with you. I'll revert this change.